code/stats/scoring.cpp

```diff
@@ -105,12 +105,10 @@ void parse_rank_tbl()
 		required_string("#End");
 
 		// be sure that all rank points are in order
-#ifndef NDEBUG
```

Comment:
Okay then, why did you break Release warning-counting in 575091aea4d0f657765b8ed2e066cb307c9f8826 ? :P